plugins/optimization-detective/class-od-url-metric-group-collection.php

```diff
@@ -73,7 +73,7 @@ final class OD_URL_Metric_Group_Collection implements Countable, IteratorAggrega
 	 * A freshness age of zero means a URL Metric will always be considered stale.
 	 *
 	 * @since 0.1.0
-	 * @var int<0, max>
+	 * @var int
```

Comment:
Would it make sense to use:

```suggestion
	 * @var int<-1, max>
```

And then throughout here.

Reply:
Changed it in b5ef506.